src/pages/CitiesPage/CitiesPage.jsx

```diff
@@ -1,17 +1,18 @@
-import NavBar from '../../components/common/NavBar/NavBar';
-import Footer from '../../components/common/Footer/Footer';
+import Page from '../../components/common/Page/Page';
+import HeroSection from './HeroSection';
+import CitiesSection from './CitiesSection';
+import NotFoundSection from './NotFoundSection';
 
 const CitiesPage = () => {
   return (
-    <>
-      <header>
-        <NavBar />
-      </header>
-      <main></main>
-      <footer>
-        <Footer />
-      </footer>
-    </>
+    <Page
+      title="Cities"
+      description="Discover comprehensive travel guides for cities around the world on Voyage. Explore detailed itineraries, local tips, and hidden gems in every city guide. Start your adventure and share your experiences with fellow travelers."
+    >
+      <HeroSection />
+      <CitiesSection />
+      <NotFoundSection />
```

Comment:
We need to render this conditionally depending on data received from BE

You can implement fake logic that imitates behaviour